automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/WorkspaceDataPage.scala

```diff
@@ -50,22 +53,37 @@ class WorkspaceDataPage(namespace: String, name: String)(implicit webDriver: Web
     * @param downloadPath the directory where the browser saves downloaded files
     * @return the relative path to the moved download file, or None if downloadPath was not given
     */
-  def downloadMetadata(downloadPath: Option[String] = None): Option[String] = {
+  def downloadMetadata(downloadPath: Option[String] = None): Option[String] = synchronized {
```

Comment:
does this do anything when there are forked JVMs?

Reply:
tests are running in parallel.